app/components/PaginationBar.tsx

```diff
@@ -77,7 +77,7 @@ export const PaginationBarComponent: React.FunctionComponent<Props> = (
         <Pagination.First onClick={() => handlePaginationByPageNumber(1)} />
         <Pagination.Prev onClick={previousTenPagination} />
         {startPage !== 1 && <Pagination.Ellipsis />}
-        <Pagination>{items}</Pagination>
+        {items}
```

Comment:
The examples for the Bootstrap Pagination component show only `<Pagination.Subcomponent>`s as children of a `<Pagination>` component - not nesting `<Pagination>` instances within each other.

Reason for change: This was causing there to be a nested `<ul>` as a direct child of another `<ul>`; the accessibility rules permit only `<li>` elements to be direct children of a `<ul>` or `<ol>`.